cli/slashingExport_test.go

```diff
@@ -0,0 +1,187 @@
+/*
+Copyright 2022 Nethermind
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cli_test
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/NethermindEth/sedge/cli"
+	"github.com/NethermindEth/sedge/cli/actions"
+	"github.com/NethermindEth/sedge/cli/actions/mock"
+	"github.com/NethermindEth/sedge/configs"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSlashingExport_ValidatorIsRequired(t *testing.T) {
+	tests := []struct {
+		name        string
+		args        []string
+		expectedErr error
+	}{
+		{
+			name:        "without flags",
+			args:        []string{},
+			expectedErr: cli.ErrInvalidNumberOfArguments,
+		},
+		{
+			name:        "with flags",
+			args:        []string{"--network", "sepolia", "--out", "slashing_data.json"},
+			expectedErr: cli.ErrInvalidNumberOfArguments,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {})
```

Comment:
Why aren't you putting the code as a subtest inside the `Run`?

Reply:
No reason to do that, that’s an error. Fixed here https://github.com/NethermindEth/sedge/pull/209/commits/0c6d8540e303f3d25c56c6e930bc65db50c93a14